scripts/requests/repositories.http

```diff
@@ -32,13 +32,35 @@ Content-Type: application/json
 {
   "revision": "main",
   "jobConfigs": {
-    "analyzer": {},
-    "advisor": {},
-    "scanner": {},
-    "evaluator": {},
-    "reporter": {},
-    "notifier": {}
-  }
+    "analyzer": {
+      "allowDynamicVersions": true,
+      "packageCurationProviders": [],
+      "skipExcluded": true
+    },
+    "advisor": {
+    },
+    "scanner": {
+    },
+    "evaluator": {
+    },
+    "reporter": {
+      "skipExcluded": true
```

Comment:
The reporter does not have a `skipExcluded` option, only the analyzer, advisor and scanner have.